tests/unit_tests/gui/test_main_window.py

```diff
@@ -134,16 +144,13 @@ def use_rundialog():
 
 
 @pytest.mark.usefixtures("use_tmpdir")
-@pytest.fixture(scope="module")
-def ensemble_experiment_has_run(opened_main_window, run_experiment, request):
+@pytest.fixture(scope="module", name="ensemble_experiment_has_run")
+def ensemble_experiment_has_run_fixture(opened_main_window, run_experiment, request):
```

Comment:
What do you think about adding types here and the other functions you are touching with this PR?

Reply:
i thought about it - it's in tests, and the types are a bit difficult to understand for me, and there's no automatic checking (mypy doesn't run on our tests), and manually i couldn't get mypy to run report on eivind's remark either, so i thought i'd keep my hands off that. i know too little about the whole gui system